ern-util/src/promptUtils.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import inquirer from 'inquirer'
+
+export async function askUserToChooseAnOption (choices: Array<string>, message?: string): Promise<any> {
```

Comment:
I like that. We have quite some calls to inquirer directly, it would be leaner to just reuse this one (and maybe other functions that will be addded to this util) where appropriate (through another "prompts" refactoring PR later on). 
Makes it also more simple to replace inquirer lib with another one as every use of it will be centralized (also ease mocking for testing) :+1: